packages/components/src/border-control/border-control-dropdown/component.tsx

```diff
@@ -240,7 +241,7 @@ const BorderControlDropdown = (
 		<Dropdown
 			renderToggle={ renderToggle }
 			renderContent={ renderContent }
-			contentClassName={ popoverClassName }
+			popoverProps={ __unstablePopoverProps }
```

Comment:
```suggestion
			popoverProps={ {
				...__unstablePopoverProps,
				className: popoverClassName,
			} }
```

The removal of the `contentClassName` breaks the components styling of the popover content to achieve the full-width reset button.

The above suggestion reinstates it but via the `popoverProps` so should be ok?

The manipulation of the popover props and the class name might be better done in the component's hook though leveraging `cx` there in case the `__unstablePopoverProps` already set a class name.

Reply:
Why we'd need to leverage cx here? I think cx is for CSS in JS no?

Reply:
Actually, I removed the need for this classname entirely.

Reply:
Apologies but I'm not sure you have removed the need for the classname just yet. The class removed in https://github.com/WordPress/gutenberg/pull/40740/commits/b75835ef0eec5eb0c2870e052b3f675cbf18aa0d removed the popover's padding so that the reset button can go full width as per its [original design](https://github.com/WordPress/gutenberg/issues/35602) and my [initial comment above](https://github.com/WordPress/gutenberg/pull/40740#discussion_r871104203).

We'll need a class to target the popover to remove the margin or a negative margin hack on the button. 

In addition, it's probably simpler to set the arbitrary width via this class short of a prop to do so on Popover. Marco had some [comments](https://github.com/WordPress/gutenberg/pull/40874#discussion_r866699667) on this on one of the alternate border control PRs.

| Trunk | This PR |
|---|---|
| <img width="297" alt="Screen Shot 2022-05-13 at 11 43 03 am" src="https://user-images.githubusercontent.com/60436221/168194625-a1b06cce-7f1a-4006-88a4-6df2ed12c021.png"> | <img width="298" alt="Screen Shot 2022-05-13 at 11 16 14 am" src="https://user-images.githubusercontent.com/60436221/168194553-edd0826f-898f-41f8-94bd-e172bcedbf97.png"> | 


> Why we'd need to leverage cx here? I think cx is for CSS in JS no?

It provides an opportunity to neatly move the logic manhandling the popover props out of the `component.tsx` while also meaning we can merge the component's default styles with a class passed via `__unstablePopoverProps`, like `classnames` basically.

Reply:
Good catch, I'll see if I can fix that.